tests/converters/test_maps.py

```diff
@@ -20,7 +20,7 @@ def test_BiMap():
     assert "BiMap(PDGID-PythiaID)" in str(bimap)
 
     with pytest.raises(MatchingIDNotFound):
-        pyid = bimap[PDGID(9000221)]
+        bimap[PDGID(9000221)]
```

Comment:
Assignment does nothing.

Reply:
That's right. I thought not assigning to a tmp variable would rather be bad practice (or else assign to `_`). But fine as you changed since the instantiation is what really matters.